slides/src/content/routing.md

```diff
@@ -114,12 +113,64 @@ For example the following code defines a link to the route at path `component-on
 <a [routerLink]="['/component-one']">Component One</a>
 ```
 
-Alternatively, you can navigate to a route by calling the `navigate` function on the router:
+Alternatively, you can navigate to a route by calling the `navigate` function on the router with the path array as the argument. For example to navigate to component-one, we could use:
+
+```javascript
+var path = ['component-one'];
+router.navigate(path);
+```
+
+---
+
+## Defining Links Between Routes (2/2)
+
+The path array can be seen as segments of an URL. 
+For example, using the following path in our array:
+
+```javascript
+var path = ['component-one', 'param1', 'param2'];
```

Comment:
In some cases we're using a preceding `/` with our path segments, sometimes not - either make this consistent, or add a few words explaining what `/` means